src/main/java/com/github/hcsp/maven/Version.java

```diff
@@ -13,5 +17,31 @@
      * @param version2 传入的版本字符串2，支持x/x.y/x.y.z，你可以假定传入的字符串一定是合法的语义化版本
      * @return -1/0/1 当version1 小于/等于/大于 version2时
      */
-    public static int compare(String version1, String version2) {}
+    public static int compare(String version1, String version2) {
+        if(version1.equals(version2)) return 0;
```

Comment:
```suggestion
        if(version1.equals(version2)) { return 0; }
```

Checkstyle没过。至于花括号的问题，推荐阅读：https://coolshell.cn/articles/11112.html

Reply:
学习了，感谢老师~